1-exercises/G-array-methods/exercise2.js

```diff
@@ -7,7 +7,8 @@
 let mentors = ["Daniel", "Irina", "Rares"];
 let students = ["Rukmini", "Abdul", "Austine", "Swathi"];
 
-let everyone; // complete this statement
+let everyone = []
+everyone = mentors + students; // complete this statement
```

Comment:
The requirement says that "  The variable everyone should be an array containing both mentors and students."

The variable that you have here is actually a string (because of the +), not an array